parser.go

```diff
@@ -95,3 +105,23 @@ func (ctx *ParserContext) parseBasicTags() {
 		}
 	}
 }
+
+func validateMetaTag(tag *MetaTag) error {
```

Comment:
HTML content may be not correct 100%. The use of this function can cause error to be returned due to a single incorrect piece of data. I suggest adding a config item to ParserCongif, says `SkipInvalidTags`, with default value `false` (to be back-ward compatible). When it is turned on, every invalid tags will be ignored instead of returning error to client.